impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -194,16 +194,12 @@ static ISize OptimumAtlasSizeForFontGlyphPairs(
     const std::vector<FontGlyphPair>& pairs,
     std::vector<Rect>& glyph_positions,
     const std::shared_ptr<GlyphAtlasContext>& atlas_context,
-    GlyphAtlas::Type type,
     const ISize& max_texture_size) {
-  static constexpr auto kMinAtlasSize = 8u;
```

Comment:
lol, thats comically small. I think this was from a time when the glyph atlas was per render pass.

Reply:
not even a single emoji!